src/LearningObjectSubmission/ElasticsearchSubmissionPublisher.ts

```diff
@@ -46,6 +46,47 @@ export class ElasticsearchSubmissionPublisher implements SubmissionPublisher {
     }
   }
 
+  /**
+   * @description
+   * In the case that an update does fail, an error will be reported for us to fix retrospectively.
+   * Until that is done, the searchable submissions will be out of date with the Learning Object data
+   * in storage.
+   * @inheritdoc
+   */
+  async updateSubmission(params: {
+    learningObjectId: string;
+    updates: Partial<LearningObject>;
+  }) {
+    const { learningObjectId, updates } = params;
+    let updateField = Object.keys(updates);
+    let updateSource: string = '';
```

Comment:
Since a default value is assigned here, TypeScript favors the convention of omitting the type annotation:

```suggestion
    let updateSource = '';
```

I don't think our current linter complains, but I figured I'd still mention it.